Cargo.toml

```diff
@@ -8,7 +8,152 @@ members = [
     "integration_test",
 ]
 
-#
+[workspace.package]
+authors = ["Centrifuge <admin@centrifuge.io>"]
+edition = "2021"
+license = "LGPL-3.0"
+homepage = "https://centrifuge.io/"
+repository = "https://github.com/centrifuge/fudge"
+
+[workspace.dependencies]
+fudge = { path = "fudge" }
+fudge-core = { path = "core" }
+fudge-companion = { path = "core/src/builder/companion" }
+fudge-test-runtime = { path = "core/src/tests/test-parachain" }
+
+scale-info = { version = "2.1.2", default-features = false, features = [
+    "derive",
+] }
+bitvec = { version = "1.0.0", default-features = false, features = ["alloc"] }
+parity-scale-codec = { version = "3.0.0", features = [
+    "derive",
+] }
+futures = "0.3.19"
+async-trait = "0.1.52"
+parking_lot = "0.12.1"
+lazy_static = "1.4.0"
+tracing = "0.1.37"
+thiserror = "1.0.30"
+tokio = { version = "1.15", features = ["macros"] }
+metered = { package = "prioritized-metered-channel", version = "0.2.0" }
+futures-timer = "3.0.2"
+integer-sqrt = { version = "0.1.2" }
+safe-mix = { version = "1.0", default-features = false }
+smallvec = "1.6.1"
+static_assertions = "1.1.0"
+hex-literal = { version = "0.2.1" }
+rustc-hex = { version = "2.0" }
+serde = { version = "1.0.102" }
+tracing-subscriber = "0.2"
+syn = { version = "1.0.83", features = ["full", "extra-traits"] }
+quote = "1.0.14"
+proc-macro2 = "1.0.36"
+proc-macro-crate = "1.3.1"
+
+polkadot-test-runtime = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-service = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-core-primitives = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-parachain-primitives = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-node-primitives = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-primitives = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-runtime-parachains = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-overseer = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+## Currently only needed to make cumulus-relay-chain-inprocess-interface compile, not sure why cumulus works though
+polkadot-cli = { git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.7.2" }
+polkadot-runtime-common = { git = "https://github.com/paritytech/polkadot-sdk", default-features = false, branch = "release-polkadot-v1.7.2" }
```

Comment:
Is it due to some macro? Sometimes macros force you to include extra dependencies to unroll correctly on the user side.

Reply:
No, the `fudge-test-runtime` is using it.